modules/oe_whitelabel_list_pages/config/install/core.entity_view_display.node.oe_list_page.full.yml

```diff
@@ -0,0 +1,134 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.full
+    - field.field.node.oe_list_page.oe_featured_media
+    - field.field.node.oe_list_page.oe_summary
+    - node.type.oe_list_page
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.full
+    - field.field.node.oe_list_page.oe_featured_media
+    - field.field.node.oe_list_page.oe_summary
+    - node.type.oe_list_page
+  module:
+    - user
+id: node.oe_list_page.full
+targetEntityType: node
+bundle: oe_list_page
+mode: full
+content:
+  'extra_field_oe_list_page_filters:node:oe_list_page':
+    settings: {  }
+    third_party_settings: {  }
+    weight: 5
+    region: content
+  'extra_field_oe_list_page_results:node:oe_list_page':
+    settings: {  }
+    third_party_settings: {  }
+    weight: 4
+    region: content
+  'extra_field_oe_list_page_selected_filters:node:oe_list_page':
+    settings: {  }
+    third_party_settings: {  }
+    weight: 2
+    region: content
+  'extra_field_oe_list_page_sort:node:oe_list_page':
+    settings: {  }
+    third_party_settings: {  }
+    weight: 3
+    region: content
+  'extra_field_oe_list_page_total_count:node:oe_list_page':
+    settings: {  }
+    third_party_settings: {  }
+    weight: 0
+    region: content
+  links:
+    settings: {  }
+    third_party_settings: {  }
+    weight: 1
+    region: content
+  search_api_excerpt:
+    settings: {  }
+    third_party_settings: {  }
+    weight: 6
+    region: content
+hidden:
+  'extra_field_oe_list_page_pager_info:node:oe_list_page': true
+  'extra_field_oe_list_page_rss_link:node:oe_list_page': true
+  langcode: true
+  oe_content_content_owner: true
+  oe_content_legacy_link: true
+  oe_content_navigation_title: true
+  oe_content_short_title: true
+  oe_featured_media: true
+  oe_summary: true
+
+    - text
+    - user
+id: node.oe_list_page.full
```

Comment:
Aren't these lines doubled in this file? `id: node.oe_list_page.full` is listed twice